### ISSUE TITLE
React.Node problem

### ISSUE THREAD

**Issue by DimitryDushkin** [brigand/babel-plugin-flow-react-proptypes#136]
React.Node (https://flow.org/en/docs/react/types/#toc-react-node) was introduced in flow 53 as universal type for react render return type. Using this type follows to such warning:
```
    WARNING in ./some-file.js
    60:48-58 "export 'Node' (imported as 'React') was not found in 'react'
```
Which is true, because this type provided only by flow.

I guess `React.Node` should be replaced by plugin to something like `PropTypes.element`?

**Comment by DimitryDushkin** [comment]
Issue resolved by replacing
```
import type { Node } from 'react';
React.Node -> Node
```

By still this type transforms to any.

**Comment by brigand** [comment]
Thanks for the report! Published as 6.0.0. Let me know if it works for you.

**Comment by SpainTrain** [comment]
I am trying to verify whether there is a regression on this in v14.  I want to ensure I understand the correct intended behavior before opening a new issue. Thanks!

Works in 13:

```js
import React from 'react'
import type {Node} from 'react'
type Props = {children: Node}
```

In 14 (same codebase only upgraded from v13 to v14) this results in the error `Error: Warning: Failed prop type: bpfrpt_proptype_Node is not defined`.

However, in v14 the following *does* work:

```js
import * as React from 'react'
type Props = {children: React.Node}
```

**Comment by brigand** [comment]
@SpainTrain that was likely fixed in 14.0.2, from [this issue](https://github.com/brigand/babel-plugin-flow-react-proptypes/issues/164).

**Comment by SpainTrain** [comment]
Thanks for quick response.

Sorry I should have noted exact versions.  I have been testing 13.0.2 and 14.0.2.  Note in the pasted error message `bpfrpt_proptype_Node`, which is a name that would only show up in 14.0.2+, I think.

I see that 14.0.3 was published, so will begin testing with that (though the change appears unrelated).

**Comment by SpainTrain** [comment]
I think the relevant line in the transpiled code is:

```js
import { bpfrpt_proptype_Node } from "react";
```

Unless react is also run through the plugin, this won't work right?

**Comment by brigand** [comment]
Ah, well there are two issues here. The import name was wrong between 14.0.0 and 14.0.3, but the current issue is that we don't detect the named import of `Node` from `'react'`, so it ends up producing this:

```js
import React from 'react';


const C = props => React.createElement('div', null);
C.propTypes = {
  children: function () {
    return (typeof bpfrpt_proptype_Node === 'function' ? bpfrpt_proptype_Node.isRequired ? bpfrpt_proptype_Node.isRequired : bpfrpt_proptype_Node : PropTypes.shape(bpfrpt_proptype_Node).isRequired).apply(this, arguments);
  }
};
import { bpfrpt_proptype_Node } from 'react';
import PropTypes from 'prop-types';
```

In 14.0.3 it should at least not error (I think), but the generated propTypes are wrong.

I should have time to fix that tonight.

**Comment by SpainTrain** [comment]
Makes sense.  That is great, thanks!

Were the `Element` and `ChildrenArray` types (https://flow.org/en/docs/react/types/) previously supported?  Once I see how `Node` is done, I would happily open a PR to add those as well.
